sdk/logs.js

```diff
@@ -31,21 +29,14 @@ const logs = async function logs(functionName, apiKey, follow, startAfter, token
   for (let attempt = 1, backoff = 3; attempt <= 3; attempt += 1, backoff *= 2) {
     try {
       // eslint-disable-next-line no-await-in-loop
-      const { statusCode, headers, body } = await rp.get(options);
-      logger.debug('logs response', { statusCode, headers, body });
+      const { body, headers } = validateResponse(await loggedRequest(options, 'get'));
```

Comment:
not as succinct, but I find
```
const response = await loggedRequest(options, 'get');
const { body, headers } = validateResponse(response);
```
more readable

Reply:
Yea I agree, it's more readable. I'll clean it up in a later patch.